pyvista/plotting/renderer.py

```diff
@@ -1263,7 +1270,10 @@ def show_bounds(
 
         # set bounds
         if bounds is None:
-            bounds = np.array(mesh.GetBounds())
+            bounds = np.array(mesh.bounds)
+        else:
+            bounds = np.asanyarray(bounds, dtype=float)
```

Comment:
This looks good to me, but perhaps it would be easier to follow the execution path if this is moved to line 1220 where a default bounds is already defined.

Reply:
Done in 27624437a5b9f6e0e8112b3aaed57200bbb6de84.